### ISSUE TITLE
jakewestfall.org link

### ISSUE THREAD

**Issue by HPCurtis** [bambinos/bambi#810]
Sorry if this isn't optimum place to alert you to issues in documentation. Howeverm the link to jakewestfall.org found in the https://bambinos.github.io/bambi/notebooks/Strack_RRR_re_analysis.html example is just sending me to a gambling website. No idea if that is because jakewestfall.org no longer exists or what.

**Comment by tomicapretto** [comment]
Thanks for letting us know. I think I've seen it in the past but I didn't realize our docs had links to that address. We'll fix it soon :)

**Comment by speco29** [comment]
https://bambinos.github.io/bambi/notebooks/Strack_RRR_re_analysis.html this is the correct link.
I can replace it with the wrong one, please let me know where to update it:)

**Comment by O-sama12** [comment]
@tomicapretto  Hi there,

Is this fix still open, if yes then let me fix the issue as I bumped here with an ample knowledge of git and Github

With regards
@Osama-12

**Comment by tomicapretto** [comment]
@O-sama12 thanks for double checking. It is not resolved indeed. Would you like to work on it? There is at least one link in that webpage to jakewestfall.org, which is a gambling site now.

**Comment by O-sama12** [comment]
Yeah and also please can you share with me the link to your discord or any communications that you and your colleagues are discussing like how to solve the problem X.

**Comment by tomicapretto** [comment]
@O-sama12 we can use the issue tracker or the discussions in GitHub

**Comment by tomicapretto** [comment]
Closing because this is not an issue anymore (I couldn't find the link)
